packages/ppx_irmin/ppx_irmin.2.5.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "Craig Ferguson <craig@tarides.com>"
+homepage: "https://github.com/mirage/irmin"
+bug-reports: "https://github.com/mirage/irmin/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mirage/irmin.git"
+
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "dune" {>= "2.7.0"}
+  "ppx_repr" {>= "0.2.0"}
+  "irmin" {with-test & post & = version}
```

Comment:
why is this `post` filter here? If the dependency is not need why is it listed? (same for `irmin-layer`)

Reply:
I thought we'd finally gotten rid of those :upside_down_face: They are indeed not necessary; I've removed them & done likewise upstream.